### PR TITLE
Add Dart logo asset and icons where appropriate

### PR DESCRIPTION
**Pull request by jonas-jonas**
@lak-proddev could you check if this works for you as well?

### CODE REVIEW

File: org.eclipse.dartboard/plugin.xml [hunk 1]
```diff
@@ -174,7 +174,7 @@
        <launchGroup
              bannerImage="platform:/plugin/org.eclipse.debug.ui/icons/full/wizban/run_wiz.png"
```
Comment (lak-proddev) [inline org.eclipse.dartboard/plugin.xml]: We need to update the appropriate image if available

File: org.eclipse.dartboard/plugin.xml [hunk 2]
```diff
@@ -174,7 +174,7 @@
...
              id="org.eclipse.dartboard.launchGroup"
-             image="platform:/plugin/org.eclipse.debug.ui/icons/full/etool16/run_exc.png"
+             image="assets/img/dart_16.png"
```
Comment (lak-proddev) [inline org.eclipse.dartboard/plugin.xml]: the eclipse conventional directory name is "icons"

File: org.eclipse.dartboard/plugin.xml [hunk 3]
```diff
@@ -134,7 +134,7 @@
         <wizard
              category="org.eclipse.dartboard.eclipse.wizards"
              class="org.eclipse.dartboard.project.DartFileWizard"
-             icon="platform:/plugin/org.eclipse.ui.ide/icons/full/etool16/newfile_wiz.png"
+             icon="assets/img/dart_16.png"
```
Comment (lak-proddev) [inline org.eclipse.dartboard/plugin.xml]: Same icon used for project and  file wizard

File: org.eclipse.dartboard/plugin.xml [hunk 4]
```diff
@@ -174,7 +174,7 @@
        <launchGroup
              bannerImage="platform:/plugin/org.eclipse.debug.ui/icons/full/wizban/run_wiz.png"
              id="org.eclipse.dartboard.launchGroup"
-             image="platform:/plugin/org.eclipse.debug.ui/icons/full/etool16/run_exc.png"
+             image="assets/img/dart_16.png"
              label="Dart Launch Group"
              mode="run">
        </launchGroup>
```
Comment (lak-proddev) [inline org.eclipse.dartboard/plugin.xml]: One other place is  
org.eclipse.dartboard.launch.LaunchConfigTab.LaunchConfigTab()
		ImageDescriptor descriptor = AbstractUIPlugin.imageDescriptorFromPlugin("org.eclipse.pde.ui", //$NON-NLS-1$
				"/icons/obj16/run_exc.png"); //$NON-NLS-1$
		image = descriptor != null ? descriptor.createImage() : null;

File: org.eclipse.dartboard/plugin.xml (hunk 1 above)
Comment (jonas-jonas) [reply org.eclipse.dartboard/plugin.xml]: Done. I added a 64x64 icon and referenced it here. Do you know where I can see the result? I couldn't find out where this icon is used.

File: org.eclipse.dartboard/plugin.xml (hunk 2 above)
Comment (jonas-jonas) [reply org.eclipse.dartboard/plugin.xml]: Done.

File: org.eclipse.dartboard/plugin.xml (hunk 3 above)
Comment (jonas-jonas) [reply org.eclipse.dartboard/plugin.xml]: I added a new icon based on the generic editor icon with a Dart logo.

File: org.eclipse.dartboard/plugin.xml (hunk 4 above)
Comment (jonas-jonas) [reply org.eclipse.dartboard/plugin.xml]: Done.

File: org.eclipse.dartboard/plugin.xml (hunk 1 above)
Comment (jonas-jonas) [reply org.eclipse.dartboard/plugin.xml]: Nevermind, found the location on the Dialog opened, when trying to run a file as Dart program, that has no launch config yet.

**Pull request merged**